core/src/main/java/tc/oc/pgm/rotation/pools/VotingPool.java

```diff
@@ -1,48 +1,42 @@
-package tc.oc.pgm.rotation;
+package tc.oc.pgm.rotation.pools;
 
-import java.util.Collections;
+import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 import java.util.concurrent.TimeUnit;
-import javax.annotation.Nullable;
 import org.bukkit.configuration.ConfigurationSection;
 import tc.oc.pgm.api.map.MapInfo;
 import tc.oc.pgm.api.match.Match;
 import tc.oc.pgm.api.match.MatchScope;
 import tc.oc.pgm.restart.RestartManager;
+import tc.oc.pgm.rotation.MapPoolManager;
+import tc.oc.pgm.rotation.vote.MapPoll;
+import tc.oc.pgm.rotation.vote.MapVotePicker;
 
 public class VotingPool extends MapPool {
 
-  // Number of maps in the vote, unless not enough maps in pool
-  public static final int MAX_VOTE_OPTIONS = 5;
-  // Number of maps required for a custom vote (/vote)
-  public static final int MIN_CUSTOM_VOTE_OPTIONS = 2;
-  // If maps were single voted, it would avg to this default
-  public static final double DEFAULT_WEIGHT = 1d / MAX_VOTE_OPTIONS;
+  // Arbitrary default of 1 in 5 players liking each map
+  public static final double DEFAULT_SCORE = 0.2;
```

Comment:
What are the advantages/disadvatages of having this score be more pessimistic (closer to 0) or optimistic (closer to 1)?

Reply:
At server start, all maps are just as likely, so there isn't a difference, however, after a vote, the scores of the maps that were voted are set to their real vote score.
What that means, is if after a vote a map gets 25% of the votes and ends up at 0.25, how will that compare against all other maps still sitting on defaults?
If the default is 0.8: the map with 25% vote is "super bad" in relation to everything else, so it's super unlikely to appear in a future vote
If the default is 0.1: the map with 25% vote is "super good" in relation to everything else, so it's super likely to appear in a future vote

From what i've seen a 0.2 is pretty typical, because there's 5 options to vote for (and it seems like players voting for multiple options is pretty rare, most people seem to not even know that's possible). So maps that get less than expectancy of 1/5th are underperforming and maps above 1/5th are overperforming, and so, the 0.2 score. This does not represent a change over the previous, its just written differently